blogposts/2022/why-fig-autocomplete-is-awesome.md

```diff
@@ -17,7 +17,7 @@ We are really excited to announce our latest [integration with Fig](https://fig.
 
 When I was onboarding at Sourcegraph one of my todos was to install and learn how to use `src`, a [command-line interface to Sourcegraph](https://docs.sourcegraph.com/cli). The installation was straightforward, but I couldn’t remember all of the different subcommands so I found myself typing `src --help` and [hitting the up arrow](https://www.commitstrip.com/en/2017/02/28/definitely-not-lazy/?) a lot.
 
-![](https://storage.googleapis.com/sourcegraph-assets/blog/042122-fig-blog-post-commit-strip.gif)
+![up arrow](https://storage.googleapis.com/sourcegraph-assets/blog/042122-fig-blog-post-commit-strip.gif)
```

Comment:
@jdorfman GitHub wouldn't let me comment on this line without making a change to it 😆 The gif is a bit distorted I think because of our CSS styling which defaults to 50% width. We can force it to be specific width but then it won't be responsive to screen size. WDYT?

Reply:
@rebeccadee I think maybe we just remove the image if it can't be responsive.